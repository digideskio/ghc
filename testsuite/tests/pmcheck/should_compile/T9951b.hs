{-# LANGUAGE OverloadedStrings #-}
{-# OPTIONS_GHC -Wall #-}

module T9951b where

f :: String -> Bool
f "ab" = True

