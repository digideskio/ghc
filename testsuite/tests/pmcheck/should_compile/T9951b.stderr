[1 of 1] Compiling T9951b           ( T9951b.hs, T9951b.o )

T9951b.hs:7:1: Warning:
    Pattern match(es) are non-exhaustive
    In an equation for ‘f’:
        Patterns not matched:
            ('a':_:_)
            (t0:_) where t0 is not one of ['a']
            []
